### DISCUSSION TITLE
Dalle-3 plugin with azure assistant doesn't work

### DISCUSSION THREAD

**Discussion by lurebat** [danny-avila/LibreChat#3114] [Troubleshooting]
### What happened?

When asking the assistant for a cat image, I get a response like this:

```
Here is the image of the cute and fluffy cat:

![Cat Image](sandbox:/mnt/data/dalle_image.png) 
```

The image isn't shown and isn't accessible	

### Steps to Reproduce

1. Create an assistant in azure
2. Connect to it in librechat
3. Add "Dalle-3" plugin in librechat to the assistant
4. Ask it to generate something

### What browsers are you seeing the problem on?

Firefox

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Working on my end: 
![image](https://github.com/danny-avila/LibreChat/assets/110412045/38dc5526-64ad-4abb-8a71-1a795ca65f7f)


How did you configure DALL-E?

**Reply by xinguaws** [reply]
> Working on my end: ![image](https://private-user-images.githubusercontent.com/110412045/341082408-38dc5526-64ad-4abb-8a71-1a795ca65f7f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4NTYwNzYsIm5iZiI6MTcyMTg1NTc3NiwicGF0aCI6Ii8xMTA0MTIwNDUvMzQxMDgyNDA4LTM4ZGM1NTI2LTY0YWQtNGFiYi04YTcxLTFhNzk1Y2E2NWY3Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNFQyMTE2MTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MTNlNjc1MjIxZDRiNmEwNDlmMmI3Njc3YjYxMTAxMzBmYmJlYWU3MWQxMjEwNjc2NDFjOGEzZGJhY2NhMDhlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.JMC7gtndC3xXpHrvEZsmS72PCGrKf5rNzf17MBheIfc)
> 
> How did you configure DALL-E?

Hi @danny-avila I am struggling to get DALL-E-3 model working, can you share the related part of the .env and librechat.yaml? I have azure's openai with dall-e-3 deployment, I can use it on the Azure's UI, really appreciate your help!

**Comment by lurebat** [comment]
Misconfiguration on my part - sorry.

**Reply by ABHIJITH-EA** [reply]
@lurebat What was your misconfiguration? Can you elaborate how did you fix the issue. I am also facing similar kind of issue. Image is not displaying.
